block-non-tracking-third-party-cookies/README.md

```diff
@@ -0,0 +1,86 @@
+# Third Party Cookies Bocking Tests
+
+Privacy Feature: https://app.asana.com/0/1198207348643509/1199093921854081/f
+
+## Goals
+
+This set of tests verifies implementation of third party non-tracker cookie blocking. In particular it focuses on verifying that:
```

Comment:
I think "third party non-tracker cookie blocking" as a separate feature is confusing. I understand that we want to test nonTracking3pCookies, but this is something that's extension only and temporary.

Let's use those tests for testing if ALL third party cookies were blocked and throw in couple of temporary tests for nonTracking3pCookies and same-entity rule (both of which are extension only).

This test set will eventually replace https://github.com/duckduckgo/privacy-reference-tests/tree/main/block-third-party-tracking-cookies . This means we should change the directory name to "block-third-party-cookies".